### PR TITLE
Add BoostIcon

### PR DESCRIPTION
**Pull request by mcellier**
## Description

Capture d’écran 2021-02-12 à 15 00 08

## What has been done

SVGOMG + use colors from branding when existing

## How it was tested

Storybook

### CODE REVIEW

File: src/icon/boostIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <defs>
+        <linearGradient
+          id="paint0_linear"
+          x1="1"
+          y1="23"
+          x2="26.084"
+          y2="18.606"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset=".049" stopColor="#2ED1FF" />
```
Comment (gabrielagabriel) [inline src/icon/boostIcon.tsx:25]: Magic color 🤷‍♀️

IconLibrary should follow Icons definition:
- One color
- Fill or Stroke
- Follow Pixar Swatches
- Respect Icon Grid: 24x24 - Baseline
....

Comment (PerrineBoissieres) [reply src/icon/boostIcon.tsx:25]: The Boost graphic identity is based on blue gradients, this has been Pixar validated and is being added into the lib as we are talking. I agree though that we could replace the color hex with a variable.

Comment (mcellier) [reply src/icon/boostIcon.tsx:25]: I didn't pull the icon from a magic basket. It's in the lib

Comment (mcellier) [reply src/icon/boostIcon.tsx]: Added a `boostBlue` color in branding

**Review by PerrineBoissieres — APPROVED** [review]

**Pull request merged**